### PR TITLE
add cta buttons on homepage

### LINKED ISSUE #597: Link to KI Landing from KlimaDAO Homepage
We want to find more opportunities to drive users to offset their carbon footprint.
The objective of this ticket is to add two CTA buttons on the homepage to encourage users to start their offset journey.

idea: add two boxes under video one for orgs > infinity, one for writing a love letter (for consumers) > love letters

Figma desgin: https://www.figma.com/file/ITpEVoi0pVYDRSLpFOaCiz/LANDINGPAGE?node-id=1659%3A598#241330708

DEFI for Organizations goes to https://www.klimadao.finance/infinity
DEFI for Individuals goes to https://www.loveletter.klimadao.finance/

### PR DESCRIPTION
**Pull request by goatsOnDrugs**
## Description
Adds the cta buttons for infinity and love letters

## Related Ticket

Resolves #597 

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

**Review by ladytrekker — APPROVED** [review]
Thanks

File: site/components/pages/Home/index.tsx [hunk 1]
```diff
@@ -175,6 +180,23 @@ export const Home: NextPage<Props> = (props) => {
           </div>
         </div>
 
+        <div className={styles.ctaButtonContainer}>
+          <ButtonSecondary
+            label={t({
+              id: "shared.infinity_cta",
+              message: "DEFI FOR ORGANIZATIONS",
```
Comment (Atmosfearful) [inline site/components/pages/Home/index.tsx:187]: Please use Normal case in the source string and use CSS to uppercase it `text-transform: uppercase`

```suggestion
              message: "DeFi for organizations",
```

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
why nothing on mobile? I think if you do a flexbox container around the two buttons with `flex-wrap: wrap` and `gap` it should work great on all screen sizes

File: site/components/pages/Home/index.tsx (hunk 1 above)
Comment (goatsOnDrugs) [reply site/components/pages/Home/index.tsx]: fixed

Comment (goatsOnDrugs) [reply site/components/pages/Home/index.tsx]: The buttons werent on the mobile view on figma so I hid them but they are displayed on mobile now.

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**